packages/input/src/Input/Input.tsx

```diff
@@ -0,0 +1,157 @@
+import { Slot as RadixSlot } from "@radix-ui/react-slot";
+import { useComposedRefs } from "@telegraph/compose-refs";
+import clsx from "clsx";
+import React from "react";
+
+import { COLOR, SIZE } from "./Input.constants";
+
+type BaseRootProps = {
+  size?: keyof typeof SIZE.Container;
+  variant?: keyof typeof COLOR.Container.default;
+  error?: boolean;
```

Comment:
Generally would prefer “hasError” or “errored” for this given it take a boolean

Reply:
Eventually for the wrapping "form control" element I was planning on having that take an `error` prop that could be a `string` or `boolean`. `string` in the case of wanting to display an error message + change colors to reflect an error state and `boolean` to change colors to accept an error state. 

Do you think it makes sense knowing that plan to make this `hasError` and leave the other as `error` or to keep them both consistent as `error`? 

Fine with either option, just want to make sure as we create more components with error states we follow the same pattern throughout.

Reply:
To me I think they should be different and the input should be “hasError” that the field component sets derived from the error string. Keeps a nice clean API on the input.

Reply:
Cool, works for me 👍